draft/spec/change-log.md

```diff
@@ -0,0 +1,108 @@
+
+#### Clarify use of registered and local extensions
+
+Change [Object Extensions](https://ocfl.io/draft/spec/#object-extensions) and [Storage Root Extensions](https://ocfl.io/draft/spec/#storage-root-extensions) to define registered extensions in terms of the [OCFL Extensions Repository](https://ocfl.github.io/extensions/). Added [Documenting Local Extensions](https://ocfl.io/draft/spec/#documenting-local-extensions) section to describe local extensions. Adds error codes [E112](https://ocfl.io/draft/spec/#E112) and [E113](https://ocfl.io/draft/spec/#E113), updates error code [E067](https://ocfl.io/draft/spec/#E067), and removes error codes `E068` and `E086` which were not being used within the community. Adds warning code [W016](https://ocfl.io/draft/spec/#W016). (Issues [#557](https://github.com/OCFL/spec/issues/557), [#565](https://github.com/OCFL/spec/issues/565))
+
+#### Improve guidance on inclusion of specification in storage root
+
+With the change from ReSpec to Markdown as the source format for the OCFL Specification it is now easy to store a complete copy of the specification in a storage root. This version suggests using the filename `ocfl_1.1.md` for a copy of the human-readable Markdown specification in the [Root Structure](https://ocfl.io/draft/spec/#root-structure) section. (Issues [#505](https://github.com/OCFL/spec/issues/505), [#554](https://github.com/OCFL/spec/issues/554))
+
+#### Fix examples to match the specification
+
+Correct several examples that in the 1.0 specification did not fully comply with the specification. (Issue [#539](https://github.com/OCFL/spec/issues/539))
+
+#### Reference RFC version of Bagit specification
+
+Update the reference to the Bagit specification from the draft <https://tools.ietf.org/html/draft-kunze-bagit-17> to [RFC8493](https://datatracker.ietf.org/doc/html/rfc8493). (Issue [#571](https://github.com/OCFL/spec/issues/571))
+
+### Corrections to validation codes
+
+#### Per-version validation codes
+
+Even for minor releases the validations codes may be updated. We have thus moved the `validation-codes.md` file into each version directory so that will be versioned along with the specification. The version of this file for the v1.1 specification is rendered as <https://ocfl.io/draft/spec/validation-codes.html>. (Issue [#553](https://github.com/OCFL/spec/issues/553))
+
+#### Fix E048 description
+
+The E048 error description in `validation-codes.md` is corrected to remove mention of `message` and `user` because they are optional. (Issue [#531](https://github.com/OCFL/spec/issues/531))
+
+#### Fix E070 description
+
+The E070 error description in `validation-codes.md` is corrected to refer to `extension` rather and `key` (which was left from an earlier draft). (Issue [#573](https://github.com/OCFL/spec/issues/573))
```

Comment:
Typo:
```
refer to `extension` rather and `key`
```
->
```
refer to `extension` rather than `key`
```

Reply:
Thanks, fixed in [e7a7b38](https://github.com/OCFL/spec/pull/610/commits/e7a7b38655b195fdc3351785dc27bed35a8f5ff0)